databricks/koalas/window.py

```diff
@@ -586,6 +588,12 @@ def _apply_as_series_or_frame(self, func):
             _, name = index_map
             new_index_map.append((SPARK_INDEX_NAME_FORMAT(len(new_index_map)), name))
 
+        if isinstance(kdf.columns, pd.MultiIndex):
```

Comment:
@itholic, all these things should be able to do it only touching metadata in Koalas at internal. Checkout `_InternalFrame(column_index=...)` at `groupby.py`

Reply:
not pretty sure, i fix the existing logic
could you please check this changing when you available whether it's right or not :)